integration/kubernetes/k8s-footloose.bats

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../.ci/lib.sh"
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	export KUBECONFIG="$HOME/.kube/config"
+	pod_name="footfed"
+	config_name="ssh-pub-key"
+	get_pod_config_dir
+
+	# Creates ssh-key
+	public_key_path="/tmp/mykey.pub"
+	key_path="/tmp/mykey"
+	sudo ssh-keygen -t rsa -N "" -f "$key_path"
+
+	# Create ConfigMap.yaml
+	sed -e "/\${ssh_key}/r ${public_key_path}" -e "/\${ssh_key}/d" \
+		"${pod_config_dir}/footloose-configmap.yaml" > "${pod_config_dir}/footloose-rsa-configmap.yaml"
```

Comment:
create variable for `"${pod_config_dir}/footloose-rsa-configmap.yaml"` as it is used several times in the script